### DISCUSSION TITLE
This is a test

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/wagtail#10992] [Q&A] [answered]
Hello, world! Testing Discussions’ "Answers enabled" UI.

**Comment by thibaudcolas** [comment]
This is my suggested answer.

**Comment by thibaudcolas** [comment]
Can I mark two answers as the answer?

**Reply by laymonage** [reply, marked as answer]
You cannot, but you can mark a reply as an answer too!

**Reply by thibaudcolas** [reply]
Great answer @laymonage
